packaging/desktop/package_darwin.sh

```diff
@@ -138,6 +138,13 @@ get_deps() {(
   curl -J -L -Ss "$installer_url" | tar zx
 )}
 
+check_ci() {(
+  temp="$(mktemp -d)"
+  (cd "$temp" && npm i github-ci-status)
```

Comment:
I think the issue is that we need this to work for prod builds too, but we don't normally do `npm i` for prod.